src/muse/utilities.py

```diff
@@ -200,14 +198,20 @@ def broadcast_techs(
     This function broadcast the first representation to the shape and coordinates
     of the second.
 
+    Note: this is not necessarily limited to `technology` datasets. For
+    example, it could also be used on a dataset of commodity prices to select prices
+    relevant to each asset (e.g. if assets exist in multiple regions). In this example,
+    installed_as_year should be set to False (see below).
+
     Arguments:
         technologies: The dataset to broadcast
         template: the dataset or data-array to use as a template
-        interpolation: interpolation method used across `year`
-        installed_as_year: if the coordinate `installed` exists, then it is
-            applied to the `year` dimension of the technologies dataset
-        kwargs: further arguments are used initial filters over the
-            `technologies` dataset.
+        installed_as_year: True means that the "year" dimension in the technologies
```

Comment:
It might be worth to add an explanation of the use case for this to be `False`, as well.